packages/user-interface/src/pages/account/account-page.mock.tsx

```diff
@@ -0,0 +1,18 @@
+import React, {ReactElement} from 'react';
+import {AccountPage} from './account-page';
+import {TestProviderWrapper} from '../../testUtils/TestProviderWrapper/TestProviderWrapper';
+import {mocksRequestBurgerGegevens} from './account-page-requests.mock';
+
+const route = '/zaken/zaak?id=82cb13cf-d2f9-4e3e-ac07-751373035ecb';
+
+const AccountPageTestComponent = ({}): ReactElement => {
```

Comment:
Is het nodig om hier een apart component van te maken. Kan deze niet gewoon toegevoegd worden aan MockAccountPage?

Reply:
Gelijk heb je.